examples/openIMIS/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "2.4"
+
+services:
+  db:
+    container_name: ${PROJECT_NAME:-openimis}-db
+    image: ghcr.io/openimis/openimis-mssql:${DB_TAG:-latest}
+    restart: always
+    environment:
+      - DB_USER_PASSWORD=${DB_PASSWORD}
+      - SA_PASSWORD=${DB_PASSWORD}
+      - DB_NAME=${DB_NAME}
+      - DB_USER=${DB_USER}
+      - ACCEPT_EULA=${ACCEPT_EULA}
+    healthcheck:
+      test: "bash /app/healthcheck.sh"
+      interval: 10s
+      timeout: 5s
+      retries: 5
+      start_period: 100s
+    networks:
+      openimis-net:
+    volumes:
+      - database:/var/opt/mssql/data
+    ports:
+      - 1433:1433
+  backend:
+    build:
+      context: https://github.com/openimis/openimis-be_py.git#Test_Harness_Setup
```

Comment:
I'd also change this branch name. Test_Harness_Setup is too generic, we should go for something related to GovStack. Also it's we shouldn't use snake_case in this context. It's suggested to use kebab-case/and-slashes for naming branches.